themes/hugo-conference/static/css/main.css

```diff
@@ -452,23 +452,34 @@ nav {
    TOPICS
    ========================================================================== */
 .topics {
-  position: relative;
```

Comment:
estos cambios son para que los distintos topics hagan "wrap" en la pantalla al hacerse más y menos ancha, y no haya tantos problemas en el móvil.

Retocadlo si queréis.

Ejemplos de nueva vista:
![image](https://user-images.githubusercontent.com/41953/101456764-cc15de80-3934-11eb-9cc7-933755594061.png)

![image](https://user-images.githubusercontent.com/41953/101456809-ddf78180-3934-11eb-8048-27f34842364a.png)